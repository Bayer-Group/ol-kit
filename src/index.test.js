import * as olKit from './index'

describe('olKit package', () => {
  it('should not export undefined', () => {
    Object.keys(olKit).forEach(exportKey => {
      expect(Boolean(olKit[exportKey])).toBe(true)
    })
  })
})
