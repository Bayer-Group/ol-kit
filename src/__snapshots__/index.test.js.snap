// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ol-kit package should not export undefined 1`] = `
Object {
  "BasemapBingMaps": [Function],
  "BasemapBlankWhite": [Function],
  "BasemapContainer": [Function],
  "BasemapManager": [Function],
  "BasemapOpenStreetMap": [Function],
  "BasemapStamenTerrain": [Function],
  "BasemapStamenTonerDark": [Function],
  "BasemapStamenTonerLite": [Function],
  "Compass": [Function],
  "ContextMenu": [Function],
  "ContextMenuCoords": [Function],
  "ContextMenuListItem": [Function],
  "ControlGroup": [Function],
  "ControlGroupButton": [Function],
  "Controls": [Function],
  "CurrentLocation": [Function],
  "GooglePlacesSearch": [Function],
  "LayerPanel": [Function],
  "LayerPanelActionExport": [Function],
  "LayerPanelActionExtent": [Function],
  "LayerPanelActionImport": [Function],
  "LayerPanelActionOpacity": [Function],
  "LayerPanelActionRemove": [Function],
  "LayerPanelActions": [Function],
  "LayerPanelBase": [Function],
  "LayerPanelCheckbox": [Function],
  "LayerPanelContent": [Function],
  "LayerPanelHeader": [Function],
  "LayerPanelLayersPage": [Function],
  "LayerPanelList": [Function],
  "LayerPanelListItem": [Function],
  "LayerPanelMenu": [Function],
  "LayerPanelPage": [Function],
  "LayerStyler": [Function],
  "Map": [Function],
  "Popup": [Function],
  "PopupActionCopyWkt": [Function],
  "PopupActionGroup": [Function],
  "PopupActionItem": [Function],
  "PopupBase": [Function],
  "PopupDataList": [Function],
  "PopupDefaultInsert": [Function],
  "PopupDefaultPage": [Function],
  "PopupPageLayout": [Function],
  "PopupPageLayoutChild": [Function],
  "PopupTabs": [Function],
  "Provider": [Function],
  "ProviderContext": null,
  "TimeSlider": [Function],
  "TimeSliderBase": [Function],
  "VectorLayer": [Function],
  "ZoomControls": [Function],
  "ZoomIn": [Function],
  "ZoomOut": [Function],
  "addMovementListener": [Function],
  "calculateViewPadding": [Function],
  "centerAndZoom": [Function],
  "connectToContext": [Function],
  "connectToMap": [Function],
  "convertFeatureToWkt": [Function],
  "convertXYtoLatLong": [Function],
  "copyTextToClipboard": [Function],
  "copyWktToClipbard": [Function],
  "createMap": [Function],
  "createSelectInteraction": [Function],
  "exportFeatures": [Function],
  "getLayersAndFeaturesForEvent": [Function],
  "getPopupPositionFromFeatures": [Function],
  "loadDataLayer": [Function],
  "removeMovementListener": [Function],
  "sanitizeProperties": [Function],
  "updateMapFromUrl": [Function],
  "updateUrlFromMap": [Function],
}
`;
